src/graphics/components/movie/storyboard.cpp

```diff
@@ -58,22 +210,54 @@ void Storyboard::script_() {
     // Phase 0
     // Introduce scene
 
-    animations_.push_back(std::make_unique<PropertyAnimation<glm::vec3>>(
-        0.0f, 2.0f, glm::vec3(0.0f, 0.0f, 10.0f), glm::vec3(0.0f, 2.0f, 5.0f),
-        movie_->object().camera().position()));
-
-    animations_.push_back(std::make_unique<PropertyAnimation<glm::vec3>>(
-        2.0f, 2.0f, glm::vec3(0.0f, 2.0f, 5.0f), glm::vec3(2.0f, 2.0f, 5.0f),
-        movie_->object().camera().position()));
+    Eigen::Matrix<float, 7, 3> path_points;
+    path_points.row(0) << 0.0f, 0.0f, 10.0f;
+    path_points.row(1) << 5.0f, -5.0f, 0.0f;
+    path_points.row(2) << 0.0f, 0.0f, -10.0f;
+    path_points.row(3) << -5.0f, 5.0f, 0.0f;
+    path_points.row(4) << 0.0f, 0.0f, 10.0f;
+    path_points.row(5) << 0.0f, 2.0f, 5.0f;
+    path_points.row(6) << 2.0f, 2.0f, 5.0f;
+    BdryConds3 bcs({bdry_cond::natural, bdry_cond::zero, bdry_cond::natural});
+    Path3 path(path_points, bcs);
+    std::vector<float> time_pts = {0.0f, 0.5f, 1.5f, 2.5f, 3.0f, 5.0f, 7.0f};
+    animations_.push_back(std::make_unique<MoveCameraAlongPath>(
+        time_pts, path, (SceneCamera3d*)&movie_->object().camera(), true));
+
+    Eigen::Matrix<float, 7, 3> look_at_points;
+    look_at_points.row(0) << 0.0f, 0.0f, 0.0f;
+    look_at_points.row(1) << 0.0f, -2.0f, -2.0f;
+    look_at_points.row(2) << 0.0f, 0.0f, 0.0f;
+    look_at_points.row(3) << 0.0f, -2.0f, -2.0f;
+    look_at_points.row(4) << 0.0f, 0.0f, 0.0f;
+    look_at_points.row(5) << 0.0f, 0.0f, 0.0f;
+    look_at_points.row(6) << 0.0f, 0.0f, 0.0f;
+    Path3 look_at_path(look_at_points);
+
```

Comment:
No I use `glm::vec3` everywhere, primarily because OpenGL is everywhere. May want to change this in the future, but for now let's just use `glm::vec`s consistently.